sdk/core/Azure.Core.TestFramework/src/InstrumentClientInterceptor.cs

```diff
@@ -26,10 +26,12 @@ public void Intercept(IInvocation invocation)
             }
 
             var type = result.GetType();
-            if (type.Name.EndsWith("Client"))
+            if (type.Name.EndsWith("Client") ||
```

Comment:
Wasn't sure if this should be removed.

Reply:
No, let's keep it, we still use this pattern in data plane